### DISCUSSION TITLE
Actividad 0 - Russo Matias

### DISCUSSION THREAD

**Discussion by MatyRusso22** [frlp-utn-ingsoft/catedra#18] [Show and tell]
https://github.com/MatyRusso22/actividad0-Russo

**Comment by RodrigoJacznik** [comment]
Hola Mati, gracias por entregar tan pronto.
Bien los mensajes de los commits.
Al querer correr la app, obtengo un error en el archivo `polls/views.py` y no la pude probar. Por favor arregla este problema para aprobar la actividad.

Saludos

**Reply by MatyRusso22** [reply]
Ahi arregle ese problema, y varios otros que no me dejaban ingresar una pregunta de manera correcta. Deberia funcionar sin complicaciones

**Reply by RodrigoJacznik** [reply]
Genial. Funciona correctamente.
La actividad esta aprobada.
